api/lib/application/authentication/oidc/oidc-controller.js

```diff
@@ -12,4 +19,57 @@ module.exports = {
 
     return h.response({ redirectLogoutUrl }).code(200);
   },
+
+  async getAuthenticationUrl(request, h) {
+    const { identity_provider: identityProvider } = request.query;
+    const oidcAuthenticationService = authenticationRegistry.lookupAuthenticationService(identityProvider);
+    const result = oidcAuthenticationService.getAuthenticationUrl({ redirectUri: request.query['redirect_uri'] });
+    return h.response(result).code(200);
+  },
+
+  async authenticateOidcUser(request) {
```

Comment:
on est dans l'oidc controller, comme ça on sera iso avec la fonction que je créé dans l'autre PR ```suggestion
  async authenticateUser(request) {
```